style.css

```diff
@@ -85,3 +89,191 @@ img {
   filter: grayscale(100%);
   gap: 20px;
 }
+
+#social_icons {
+  padding-left: 8px;
+}
+
+.social_media h3 {
+  margin: 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+  row-gap: 100px;
+  margin-top: 42px;
+  margin-bottom: 114px;
+}
+
+.card {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  min-height: 90%;
+  padding: 16px;
+  margin: 0 24px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Good job so far, your project card design is good but there are a few things you need to work on. From the project requirement, your work section needs to have the margins (top, bottom, left, right) as defined in the chosen design :smile:  Please fix it in order to make it similar to the Figma template. Please check the screenshots below :arrow_down: to see the differences. 

#### Your design
![Screenshot from 2022-10-28 14-48-03](https://user-images.githubusercontent.com/6615394/198672839-06649d78-ab12-4c88-bed3-e230a30fb556.jpg)

#### Figma template
![Screenshot from 2022-10-28 14-46-59](https://user-images.githubusercontent.com/6615394/198672874-b071c2d2-9f3c-41a3-8544-4e7b85548056.png)

- [ ] Please make sure to have the same background template for your work section background color `grey` in order to make it similar to the Figma template :+1: